excercises/2-locators/locators-element-all.spec.ts

```diff
@@ -5,33 +5,24 @@ describe('Protractor workshop app', function () {
 	beforeAll(async function () {
 		await browser.get("/jak-to-zrobic-w-js");
 	});
-
-	/**
-	 * Check http://angular.github.io/protractor/#/api?view=ElementArrayFinder
-	 * to see how get function can be used
-	 */
 	it('should have menu items with links to "Home", "About", "Services", "Blog", "Contact",  pages', async function () {
 		const menuText = ["Home", "About", "Services", "Blog", "Contact"]
-		await browser.get("/jak-to-zrobic-w-js");
 		var menuItems = element.all(by.css('ul.nav > li > a'));
 		expect(await menuItems.get(0).getText()).toEqual(menuText[0]);
```

Comment:
Why you checking here only one menuItem instead of whole list? Expect is not equal to the title of this it.